include/lc3_private.h

```diff
@@ -173,4 +173,15 @@ struct lc3_decoder {
     }
 
 
+/**
+ * Change the visibility of interface functions
+ */
+
+#ifdef _WIN32
+#define LC3_EXPORT __declspec(dllexport)
```

Comment:
i wonder if LC3_EXPORT could be moved to the src functions rather than a header which is public?

Reply:
I see, thanks